icicle/backend/cpu/src/curve/cpu_msm.hpp

```diff
@@ -326,11 +378,29 @@ template <typename A, typename P>
 void Msm<A, P>::run_msm(
   const scalar_t* scalars, const A* bases, const unsigned int msm_size, const unsigned int batch_idx, P* results)
 {
-  phase1_bucket_accumulator(scalars, bases, msm_size);
+#ifdef LOG_UTILIZATION
+  Timer tmsm("Total msm time");
+#endif
```

Comment:
you can share the ifdef which is already not that good looking

Reply:
I don't understand